broker/src/test/java/io/zeebe/broker/transport/backpressure/PartitionAwareRateLimiterTest.java

```diff
@@ -28,11 +29,22 @@
 
   @Before
   public void setUp() {
-    IntStream.range(0, PARTITIONS).forEach(i -> partitionedLimiter.addPartition(i));
+    IntStream.range(0, PARTITIONS).forEach(partitionedLimiter::addPartition);
   }
 
   @Test
   public void shouldPartitionsHaveItsOwnLimiter() {
```

Comment:
I do not understand this test or the method name. I don't see the connection between method name, given, when and then

Reply:
I renamed and refactored it.